### PR TITLE
'Send' button is shown for watch-only collectibles and can navigate to the 'send' flow #19743

### LINKED ISSUE #19743: 'Send' button is shown for watch-only collectibles and can navigate to the 'send' flow
Can be taken after merge of https://github.com/status-im/status-mobile/pull/19515

### Step:
1. Add watch only account with some collectibles (example: `0x8d2413447ff297d30bdc475f6d5cb00254685aae`. Take a look some collectibles are not fetched until relogin due existing issue https://github.com/status-im/status-mobile/issues/18613 )
2. Open watch only collectible

### Actual result:
'Send' button is shown for watch-only collectibles and can navigate to the 'send' flow
image

### Expected result:
Watch-only collectible page is shown as per design without 'send' button
image

### NOTE: 
if user has a particular multicollectible in both a watch-only and a regular account, then the following drawer should be shown as it is for regular accounts.
https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=10771-86917&mode=design&t=9CFi8hGhRdaoLkDM-0

Discussed with the design team [here](https://discord.com/channels/624634427930312714/1228371926871703552/123082076373765735) that this logic is applied to assets, and I believe it can also be applied to collectibles

### OS: 
IOS, Android

### Devices:
- Pixel 7a, Android 13
- iPhone 11 Pro Max, IOS 17

### PR DESCRIPTION
**Pull request by mmilad75**
fixes #19743 

Screenshot 2024-05-06 at 17 55 49

### CODE REVIEW

**Review by FFFra — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

File: src/status_im/contexts/wallet/collectible/view.cljs [hunk 1]
```diff
@@ -28,15 +28,16 @@
      collection-name]]])
 
 (defn cta-buttons
-  [chain-id token-id contract-address]
+  [chain-id token-id contract-address watch-only?]
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/collectible/view.cljs]: nit, probably better to make this a map now

**Comment by VolodLytvynenko** [comment]
Hi @mmilad75 thank you for PR. No issues from my side. Ready to be merged

File: src/status_im/contexts/wallet/collectible/view.cljs (hunk 1 above)
Comment (mmilad75) [reply src/status_im/contexts/wallet/collectible/view.cljs]: Sure 👍

**Comment by VolodLytvynenko** [comment]
Hi @mmilad75 thank you for PR. No issues from my side. PR is ready to be merged

**Pull request merged**